docs/docs/code/python/README.md

```diff
@@ -0,0 +1,279 @@
+# Writing Python in Code Steps
+
+**Anything you can do in Python can be done in a Pipedream Workflow**. This includes using any of the [350,000+ PyPi packages available](https://pypi.org/) in your Python powered workflows. 
+
+Pipedream supports [Python v{{$site.themeConfig.PYTHON_VERSION}}](https://www.python.org) in workflows.
+
+::: warning
+Python steps are available in a limited alpha release.
+
+You can still run arbitrary python code, including [sharing data between steps](/code/python/#sharing-data-between-steps) as well as [accessing environment variables](/code/python/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
+:::
+
+
+## Adding a Python code step
+
+1. Click the + icon to add a new step
+2. Click "Custom Code"
+3. In the new step, select the Python language runtime in language dropdown
```

Comment:
Bold **Python**

Reply:
Actually, to make it closer to the real formatting of the option in the dashboard, I think it makes the most sense to say `python`